server.js

```diff
@@ -178,9 +178,13 @@ let postUser = (request, response) => {
   readIncoming(request, (incoming) => {
       let user = insertsValuesObject(JSON.parse(incoming));
       console.log(user);
+      // let token = createToken(user);
       createUserDb(user)
-        .then((data) => response.end('Created user!'))
-        .catch(error => {console.log(error)});;
+        .then((data) => db.query(`SELECT username, password, userid from users where
```

Comment:
Might be nicer to break out the query into a function.  I think we will use it again and keeps all the DB manipulation stuff together.

Reply:
sounds good. will do.